docs/src/conf.py

```diff
@@ -0,0 +1,31 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'metatensor-models'
+copyright = '2023, metatensor-models developers'
+author = 'metatensor-models developers'
+release = '2023.11.29'
```

Comment:
I would parse everything from the `pyproject.toml` to not have this twice. You can take a look at [skmatter](https://github.com/scikit-learn-contrib/scikit-matter/blob/916c37f3e9d069d26923a7dced840d3b762e61a2/docs/src/conf.py#L13-L39) for some inspiration.